app/pages/Auction/BidActionPanel/BidNow.js

```diff
@@ -408,7 +418,7 @@ function getTotalBids(domain) {
   for (const {bid} of domain.bids) {
     if (bid.own) {
       // This is our bid, but we don't know its value
-      if (!bid.value)
+      if (bid.value == null)
```

Comment:
`===`?

Reply:
The reason is to catch both `undefined` and `null`:

```
> a = {a:1}
{ a: 1 }
> a.b
undefined
> a.b === null
false
> a.b == null
true
```

I picked up this coding style from the API modules in bcoin/hsd.